### DISCUSSION TITLE
Koreader app directly enters the book interface instead of the folder interface when opening the software.

### DISCUSSION THREAD

**Discussion by wede3689129** [koreader/koreader#10367] [Ideas]
Because I often read only one book at a time, I have to open the book in the folder every time I enter the APP, so I hope there is an option to open the book I read before directly when I enter the APP, saving the trouble of selecting books from the folder.

**Comment by Frenzie** [comment]
See here:
<img src=https://user-images.githubusercontent.com/202757/235295869-bc30ce45-a77d-47f7-9f5c-a4d78fc65a2e.png width=40%>

A minor issue, it might show up as nil until you click on it. See https://github.com/koreader/koreader/issues/10368

**Reply by wede3689129** [reply]
As you can see, I am using the version of the Android app. Clicking on File Settings will only allow me to enter the file browser, and I cannot set the option to immediately access the last book I watched when the app is launched. Perhaps it's because I haven't fully understood your meaning or there may be an issue with the version I am using. I hope to receive your reply.
![IMG_20230504_011257](https://user-images.githubusercontent.com/52503283/235992678-2165ec8d-3898-4908-85eb-699acd57f508.jpg)
![Screenshot_20230504_011306048_KOReader](https://user-images.githubusercontent.com/52503283/235992796-a7971cd9-1eaf-4cc8-81cb-4b125abe2612.jpg)

**Reply by Frenzie** [reply]
It's the menu in the file browser you need, not in the reader.
